src/slave/containerizer/mesos/isolators/cgroups/subsystems/blkio.cpp

```diff
@@ -107,230 +108,240 @@ Future<ResourceStatistics> BlkioSubsystemProcess::usage(
   CgroupInfo::Blkio::CFQ::Statistics totalCfqRecursive;
   CgroupInfo::Blkio::Throttling::Statistics totalThrottling;
 
-  // Get CFQ statistics.
-  Try<vector<cgroups::blkio::Value>> time = cfq::time(hierarchy, cgroup);
-  if (time.isError()) {
-    return Failure(time.error());
+  // Get CFQ statistics, if available - CFQ was removed from kernel 5.0.
```

Comment:
Can you please add https://github.com/torvalds/linux/commit/f382fb0bcef4c37dc049e9f6963e3baf204d815c in  this comment for more context?

Reply:
Done.